docs/administrator/identity/README.md

```diff
@@ -14,11 +14,11 @@ The following roles are available with a default installation:
 
 | Role | Description |
 | ---- | ----------- |
-| `ACTIVITI_ADMIN` | Provides access to the [Administrator Application](../README.md). Users require this role to be given administrator access to an application. If a user has administrator access to an application they will be able to see the **Process Admin** functions meaning they can monitor process instances and tasks and view the audit log |
+| `ACTIVITI_ADMIN` | Provides access to the [Administrator Application](../README.md). Users require this role to be given administrator access to an application.  |
 | `ACTIVITI_DEVOPS` | Provides access to the [Administrator Application](../README.md). Users with this role will be able to see the **DevOps** functions meaning they can deploy projects, create deployment descriptors and monitor applications |
 | `ACTIVITI_IDENTITY` | Provides access to the [Administrator Application](../README.md). Users with this role will be able to see the **Identity** functions meaning they can manage users, groups and roles |
```

Comment:
Not sure about this one. What do we mean with Administrator Application? 
The identity has a different section called Identity

Reply:
Updated the description after your explanation 👍